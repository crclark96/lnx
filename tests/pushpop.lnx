=rax;
ebx=;
=ah =al;
bl= bh=;
