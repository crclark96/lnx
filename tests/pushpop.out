push rax
pop ebx
push ah
push al
pop bl
pop bh
